js/ui/sideComponent.js

```diff
@@ -14,7 +14,7 @@ const SIDE_COMPONENT_ROLE = 'eos-side-component';
 /**
  * isSideComponentWindow:
  * @metaWindow: an instance of #Meta.Window
- * @return: whether the #Meta.Window belongs to a #SideComponent 
+ * @return: whether the #Meta.Window belongs to a #SideComponent
```

Comment:
Ah - I separated those whitespace fixes out into separate commits. But I suppose we don't want them, so I dropped the commits entirely.